ArmPkg/Drivers/ArmGicDxe/GicV3/ArmGicV3Dxe.c

```diff
@@ -688,7 +688,7 @@ GicV3DxeInitialize (
   ArmGicV3SetPriorityMask (0xff);
 
   // Use combined priority drop and deactivate (EOImode == 0)
-  RegValue  = ArmGicV3GetControlRegister ();
+  RegValue  = (UINT64)ArmGicV3GetControlRegister ();

```

Comment:
Is this cast needed? Presumably GCC doesn't need it, if it is already checked in, and we only do MSVC for ARM64 where UINTN is UINT64, so does it actually complain without this?

Reply:
dropped